docs/blog/vitest-3.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Vitest 3.0 is out!
+author:
+  name: The Vitest Team
+date: 2025-01-17
+sidebar: false
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: Announcing Vitest 3.0
+  - - meta
+    - property: og:image
+      content: https://vitest.dev/og-vitest-3.jpg
+  - - meta
+    - property: og:url
+      content: https://vitest.dev/blog/vitest-3
+  - - meta
+    - property: og:description
+      content: Vitest 3.0 Release Announcement
+  - - meta
+    - name: twitter:card
+      content: summary_large_image
+---
+
+# Vitest 3.0 is out!
+
+_January 17, 2025_
+
+![Vitest 3 Announcement Cover Image](/og-vitest-3.jpg)
+
+We released Vitest 2 half a year ago. We have seen huge adoption, from 4,8M to 7,7M weekly npm downloads. Our ecosystem is growing rapidly too. Among others, [Storybook new testing capabilities powered by our vscode extension and browser mode](https://storybook.js.org/docs/writing-tests/test-addon) and Matt Pocock is building [evalite](https://www.evalite.dev/), the first GenAI-powered test tool, on top of Vitest.
```

Comment:
cc. @mattpocock

Reply:
"Matt Pocock is building [Evalite](https://www.evalite.dev/), a tool for evaluating AI-powered apps, on top of Vitest.